projects/jdwp/codegen/new_type_generator.py

```diff
@@ -0,0 +1,21 @@
+"""New type Generator."""
+from projects.jdwp.defs.schema import PrimitiveType
+
+
+def get_python_type(jdwp_type: str) -> str:
+    """Map JDWP type to Python type."""
+    mapping = {
+        "string": "str",
+        "int": "int",
+        "byte": "bytes",
+        "boolean": "bool",
+        "dict": "dict",
+    }
+    return mapping.get(jdwp_type, "str")
+
+
+def generate_new_types():
```

Comment:
great ! Could you factor out code computing the definition of the type alias to a separate function and add a test for it ?